### ISSUE TITLE
Background thread/ Use Handler for saving, sharing and retrieving photos

### ISSUE THREAD

**Issue by MadelynTav** [tashsmit/Meme5#27]
To prevent the app from skipping frames due to too much activity on main thread

**Comment by AnthonyFermin** [comment]
Good idea, this is a big issue in the app and using a background thread and handlers will definitely improve the skipping frames issues we have. Thanks!
